src/components/Popover/PeoplePickerFilterPopover.tsx

```diff
@@ -0,0 +1,59 @@
+import { FunctionComponent, useContext, useState } from "react";
+import { Form } from "react-bootstrap";
+import { Placement } from "react-bootstrap/esm/Overlay";
+import { IPerson, Person } from "../../api/DomainObjects";
+import { FilterValue } from "../../api/ProcessesApi";
+import { UserContext } from "../../providers/UserProvider";
+import { PeoplePicker } from "../PeoplePicker/PeoplePicker";
+import { FilterPopover } from "./FilterPopover";
+
+
+export interface PeoplePickerFilterPopoverProps {
+    show: boolean,
+    target: any,
+    titleText: string,
+    placement?: Placement,
+    onSubmit: (filterValue: FilterValue) => void,
+    clearFilter(): void,
+    handleClose: () => void
+}
+
+export const PeoplePickerFilterPopover: FunctionComponent<PeoplePickerFilterPopoverProps> = (props) => {
+
+    const [person, setPerson] = useState<IPerson>();
+    const { user } = useContext(UserContext);
+
+    const onSubmit = () => {
+        if (person) {
+            props.onSubmit(person)
+        }
+    }
+
+    const onClear = () => {
+        setPerson(undefined);
+        props.clearFilter();
+    }
+
+    const setUser = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        e.preventDefault();
+        setPerson(user);
+        return false;
+    }
+
+    return (
+        <FilterPopover {...props} onSubmit={onSubmit} clearFilter={onClear}>
+            <Form>
+                {user !== undefined &&
+                    <div className="mb-2"><button className="link-button" onClick={setUser}>Filter by Me</button></div>
```

Comment:
Any reason you're using custom css instead of the React-Bootstrap link variant for a button?
<Button variant="link">Link</Button>

Reply:
I didn't like the way Bootstrap's link button looked. It still looks like a button when you click on it.